test/verify/check-machines

```diff
@@ -2587,6 +2587,7 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         m = self.machine
 
         self.startVm("subVmTest1")
+        self.machine.execute("touch ~/phonycdrom; virsh attach-disk subVmTest1 ~/phonycdrom sdb --type cdrom --config")
```

Comment:
Test failed with 

```
20:01:33.030223Z qemu-system-x86_64: -blockdev {"driver":"file","filename":"/root/phonycdrom","node-name":"libvirt-1-storage","auto-read-only":true,"discard":"unmap"}: Could not open '/root/phonycdrom': Permission denied
10-12T20:01:33.030223Z qemu-system-x86_64: -blockdev {"driver":"file","filename":"/root/phonycdrom","node-name":"libvirt-1-storage","auto-read-only":true,"discard":"unmap"}: Could not open '/root/phonycdrom': Permission denied
```
 in the journal. I just tried this but instead of using `/root/phonycdrom`, using `/tmp/phonycdrom`, and that worked fine. I think it does have to be created in a place where the admin user could access it?

Reply:
Yes thanks. But in that case, AppArmor would complain. Apparently it expects only several disk locations https://bugs.launchpad.net/ubuntu/+source/libvirt/+bug/1803742 .
I tried changing the location to /var/lib/libvirt/images/phonycdrom, lets see